mantidproject/mantid
# Indirect Data Analysis - post execution plot and save

The plotting and saving of mantid workspaces has been refactored to the interface and push buttons for plotting and saving post algorithm executions have been implemented for all IndirectDataAnalysis tabs.

 **To Test:**
Files to test with: \olympic\babylon5\Public\Louise McCann\DataAnalysis
For each tab load relevant files and run algorithm. Check each plot option plots a suitable graph and that the appropriate workspaces have been saved in your save directory.
- [ ] Elwin
- [ ] MSD Fit
- [ ] I(Q, t)
- [ ] I(Q, t) Fit 
- [ ] JumpFit

Fixes #17022 .



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
